### PR TITLE
Initial support for journals service

### PR DESCRIPTION
**Pull request by bill-filler**
Initial PR for deploying new Journals IDA in sandbox.
---
Journals needs elasticsearch5.6 so runs it in a docker container as to not conflict with existing elasticsearch1.5 installed and used by discovery/forumns

See related PR to add journals checkbox and version to jenkins:
https://github.com/edx/jenkins-job-dsl/pull/364

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

**Comment by bill-filler** [comment]
@fredsmith 
Fred can you please do an initial review of this PR. Thanks

File: playbooks/roles/journals/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    owner: "{{ common_web_user }}"
+    group: "{{ journals_user }}"
+    mode: "0775"
+  with_items:
+    - "{{ JOURNALS_MEDIA_ROOT }}"
+  tags:
+    - install
+    - install:base
+
+- name: build elasticsearch5 image
```
Comment (fredsmith) [inline playbooks/roles/journals/tasks/main.yml:36]: Can you tag these tasks?

**Review by fredsmith — APPROVED** [review]
One minor nit - please tag your tasks.  Otherwise looks good.

File: playbooks/roles/edxapp/tasks/deploy.yml [hunk 2]
```diff
@@ -441,6 +441,16 @@
     - manage
     - manage:db
 
+- name: create catalog integration
+  shell: '{{ edxapp_venv_bin }}/python ./manage.py lms --settings={{ edxapp_settings }} --service-variant lms create_catalog_integrations --enabled --internal_api_url "{{ EDXAPP_DISCOVERY_API_URL }}" --service_username "{{ DISCOVERY_SERVICE_USER_NAME }}"'
+  args:
+    chdir: "{{ edxapp_code_dir }}"
+  become_user: "{{ common_web_user }}"
+  when: true
```
Comment (feanil) [inline playbooks/roles/edxapp/tasks/deploy.yml:449]: I think you need a different when condition here.  This breaks the edxapp AMI builds.

Comment (bill-filler) [reply playbooks/roles/edxapp/tasks/deploy.yml:449]: @feanil 
will fix that to conditionalize and test it

**Pull request merged**